languages/common-lisp/exercises/concept/lists/.meta/design.md

```diff
@@ -0,0 +1,66 @@
+This issue describes how to implement the `lists` concept exercise for the Common Lisp track.
+
+## Getting started
+
+**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:
+
+- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
+- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
+- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)
+
+Please also watch the following video:
+
+- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).
+
+## Learning objectives
+
+- Be able to create a list using quote (`'`), `list`, and `cons`
+- Know that lists are constructed from `cons` cells terminated by `NIL`
+- Be able apply `car` / `first` and `cdr` / `rest` to get the parts of a list
+- Know about `nth` and `second`, `third` ... `tenth` for accessing elements of a list
+- Be able to use a list like a stack with `push` and `pop`
```

Comment:
I don't think we have `push` or `pop` right now, and that's probably for the best, but we should move it to something like out of scope, I suppose